containers/kubernetes/reference-content/secure-cluster-with-private-network.mdx

```diff
@@ -0,0 +1,127 @@
+categories:
+  - kubernetes
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+Scaleway Kubernetes Kapsule provides a managed environment to create, configure and run a cluster of preconfigured machines for containerized applications. This allows you to create Kubernetes clusters without the complexity of managing the infrastructure.
+
+All new Kubernetes clusters are deployed with a Scaleway regional Private Network. **We recommend migrating existing clusters to a regional Private Network as well, using the Scaleway console.**
+
+Currently, worker nodes are still shipped with public IP addresses.
+These IPs are used for outgoing traffic from your nodes towards the Internet only; no services are listening on them by default.
+Despite having public IP addresses for limited maintenance and operational reasons, you can be assured that your cluster remains fully secure. Find more details below.
+
+<Lightbox src="kubernetes-kapsule-on-pn-v2.webp" alt="Schema Kubernetes Kapsule on Private Networks" size="large" />
+
+## Why have a Private Network for your Kubernetes Kapsule cluster?
+
+A Private Network brings additional features to your cluster, such as:
+* Implementation of best practices in terms of security: All Scaleway resources to communicate securely (Instances, Load Balancers, Managed Databases), with less surface of attacks
+* Compliance with expectations from the market (enterprise customers)
+* Less manual configuration work such as security group configuration, IP range configuration etc.
+* Lower latency
+
+You can create new regional Private Networks via the Scaleway Console and quickly add clusters to existing regional Private Networks using the Scaleway Console. Each cluster is autoconfigured using a /22 IP subnet thanks to Scaleway’s IPAM.
+
+## Why do worker nodes still have public IPs?
+
+Public IP addresses are currently required to maintain the administration of your worker nodes and their control plane to ensure the full functionality of your clusters. 
+Do note, however, that incoming traffic on these public IPs is blocked by default thanks to a security group set at DROP ALL, and only outgoing traffic necessary for Kubernetes to run (e.g., apt, Docker image pull, etc.) is allowed. 
+
+## Can I use a Public Gateway with my Private Network to exit all outgoing traffic from the nodes?
+
+At this moment, outgoing traffic is through the public IP of each node.
+Public Gateway will soon be compatible with IPAM and thus supported by Kapsule in a future release.
+
+## How can I migrate my existing clusters to regional Private Networks?
+
+You can use the Scaleway console to migrate all your previous clusters (recommended).
+
+1. In Kubernetes, locate your public clusters in the list (Network column)
```

Comment:
I think these steps could be clearer, not sure what is meant by "Network column"?